frontend/src/users/api/subscribe.ts

```diff
@@ -0,0 +1,16 @@
+import { fetchEventSource } from '@microsoft/fetch-event-source';
+
+export const userAbortController = new AbortController();
+
+export const subscribe = (jwt: string): Promise<void> => {
```

Comment:
To chyba do modulu notyfikacji. I zastanawiam sie, czy zamiast `Promise<void>` nie powinno byc `Promise<AbortController>`. Albo w ogole samo `AbortController`, I ten abort controller bym tworzyl w ciele funkcji, a nie na zewnatrz.